CentralisedPackageConverter/OutputExtensions.cs

```diff
@@ -8,7 +8,7 @@ namespace CentralisedPackageConverter;
 public static class Output
 {
     public static void InfoLine(string message) => InfoLine(message, Array.Empty<object>());
-    public static void InfoLine(string message, params object[] args) => AnsiConsole.MarkupLine(Markup.Escape(message), args);
+    public static void InfoLine(string message, params object[] args) => AnsiConsole.MarkupLine(message, args);
```

Comment:
This change was required because the `[green](project file path[/]` markup wasn't being rendered.